creator-node/src/routes/tracks.js

```diff
@@ -691,7 +692,7 @@ module.exports = function (app) {
           }
         }
       } catch (e) {
-        req.logger.error(`Error looking for stream fallback in redis`, e)
+        req.logger.error({err: e}, 'Error looking for stream fallback in redis')
```

Comment:
nit - could actually call this `error` :)

Reply:
How is this latest update? Or did you want `err` to also be named `error`? Wasn't sure, so I just renamed `e` to `error`.